scripts/ci/copyright_check.py

```diff
@@ -28,6 +30,19 @@ def get_header(has_shebang, regexes):
     indices = range(init_idx, init_idx + len(regexes))
     return zip(regexes, indices)
 
+def match_somewhere(regexes, lines, empty_or_comment_regex):
+    """ Matches all MODIFIED_HEADER patterns within the file. This is used
+    when the license is not at the top header, and there are licenses of
+    external libraries. """
+    maybe_match_head_index = [index for index, line in enumerate(lines) if regexes[0].search(line)][:1]
+    if maybe_match_head_index and maybe_match_head_index[0] + len(regexes) <= len(lines):
+        match_head_index = maybe_match_head_index[0]
+        matches_pre_license = [empty_or_comment_regex.search(lines[index]) for index in range(match_head_index)]
```

Comment:
nit: prefer using "()" instead of "[]". This use case is not that bad since it's creating a list of boolean.

Reply:
https://github.com/model-checking/kani/pull/1447/commits/361a8975a28dad766354549cb876697e584821b4